README.rst

```diff
@@ -87,9 +116,11 @@ version of IPython and its dependencies.
 Install
 =======
 
-This package uses distutils. To install, execute as usual::
+This package uses distutils. To install, execute as usual:
```

Comment:
I recommend removing the references to distutils and just state that it can be installed using `pip install scikit-aero`, and `pip install --editable .` for local installs :blush:

The reason is that `python setup.py xxx` is supposed to go away... Soon! The [PyPA](https://github.com/pypa/) is standardizing around stuff that makes `setup.py` files disappear.